react-finger-exercises/react/ecmascript/01 - spread/index.js

```diff
@@ -1,10 +1,29 @@
 import { isArray } from './utils';
 
-export function min() {
-  isArray();
-  return Math.min();
+export function min(...args) {
+  if (arguments.length === 0) {
+      return undefined;
+  } else if (isArray(...args)) {
+      const [list] = args;
+      return Math.min(...list);
```

Comment:
I would define only one `return Math.min(...list)`, assigning different values to `list`, depending if args is an array or a list